kolibri/core/assets/src/views/icons/KLabeledIcon.vue

```diff
@@ -1,22 +1,33 @@
 <template>
 
   <span class="labeled-icon-wrapper">
-    <div class="icon">
+    <div ref="icon" class="icon">
```

Comment:
i.e. something like:

| topic view | content view |
|--|--|
| ![image](https://user-images.githubusercontent.com/2367265/54857608-8c6d1600-4cbd-11e9-925d-39e91e65b009.png) | ![image](https://user-images.githubusercontent.com/2367265/54857607-870fcb80-4cbd-11e9-83a2-e00c1c2f2ca6.png) |

* parent back links always above header, always purple clickable
* in-progress icons top-right in both cards and pages
* type icons top-left in both cards and pages

Reply:
final question: should we be wrapping the content in a `KPageContainer`? They still look a little naked with the grey background:

> ![image](https://user-images.githubusercontent.com/2367265/54857745-2e8cfe00-4cbe-11e9-82fb-a77c19d33c28.png)

Reply:
There's a separate issue for the KPageContainer here #5242. There are multiple pages where it's not used as well, so it would be nice if it were all fixed in one patch.

Reply:
Same with the header+icon pairings in #5119. Should try to fix this holistically.

Reply:
ooooh, @jonboiser just brought up a great point.

I _think_ we'd decided to start going with the convention that in full-screen windows, we'd do:

* × [current thing]
* ← [parent thing]

In other words, back buttons are labeled by what you're going back to, and close buttons are labeling the thing you're about to close. Is that right @khangmach?

In that case, everything becomes much simpler:

> ![image](https://user-images.githubusercontent.com/2367265/54858479-29ca4900-4cc2-11e9-8860-f5b533ce21ae.png)

Reply:
@indirectlylit Yes we can just use the implementation above. Also, is the gold star floating right an updated implementation? Haven't seen that yet

Reply:
The only changes I'm going to make are to

1. Make the label to the "<-" the parent topic name (if in browse mode)
1. Make the label to the "<-" the lesson name (if in lesson mode)
1. Remove the topic back link.

Moving icons around is a broader design system issue that should maybe be addressed in #5119

Reply:
> Also, is the gold star floating right an updated implementation? 

that was a recommendation to mirror the layout of content cards:

![image](https://user-images.githubusercontent.com/2367265/54969886-40c89f80-4f3d-11e9-8c09-57e31a5d7a5d.png)

and also to be a bit more similar to coach reports, which put the progress info in a separate column rather than inline with the title